pattern_import_export/tests/common.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2020 Akretion France (http://www.akretion.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from ..models.ir_exports import COLUMN_M2M_SEPARATOR
+from odoo.addons.queue_job.tests.common import JobMixin
+
+
+class ExportPatternCommon(JobMixin):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
+        cls.Exports = cls.env['ir.exports']
```

Comment:
by curiosity does there is some guideline (in oca) with this syntax?

Reply:
I didn't see it (for now there is both). But IMO, I don't like doing `cls.export_obj = cls.env['ir.exports']`.
I think when we load a `Model`, it's considered like a `class` so by convention, the name should be Uppercase to know that's a `class` (but maybe I'm wrong).